frontend/packages/console-shared/src/components/markdown-extensions/MarkdownCopyClipboard.tsx

```diff
@@ -62,6 +62,19 @@ const MarkdownCopyClipboard: React.FC<MarkdownCopyClipboardProps> = ({
   rootSelector,
 }) => {
   const elements = docContext.querySelectorAll(`${rootSelector} [${MARKDOWN_COPY_BUTTON_ID}]`);
+  const forceRender = useForceRender();
+  /**
+   * During the first render of component docContext is not updated with the latest copy of `document`
+   * because we use `dangerouslySetInnerHTML` to set the markdown markup to a div.
+   * So while react updates the dom with latest of markdown markup with `dangerouslySetInnerHTML`
+   * this component mounts and access the old copy of dom in which there are no elements present
+   * with querySelector.
+   * So using forceRender to rerender the component and access the `document` in which markdown markup is present.
+   */
+  React.useEffect(() => {
+    forceRender();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
You should update `InlineMarkdownView` instead to delay the rendering of the extensions by one render cycle.